### PR TITLE
Create Day3.md

### PR DESCRIPTION
**Pull request by whtsogreat**

### CODE REVIEW

File: Learning/Day3.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
+https://www.navicat.com/en/download/navicat-premium#mac
+
+Mysql：
+
+作业 
+https://book.douban.com/subject/3354490/
+
```
Comment (weiwensangsang) [inline Learning/Day3.md:7]: no goods.

File: Learning/Day3.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+SELECT prod_name FROM products;
+
+SELECT prod_id, prod_name, prod_price FROM products;
+
+SELECT * FROM products;
+
+SELECT DISTINCT vend_id FROM products; 只返回不同(唯一)的
+vend_id行;
+
+SELECT prod_name FROM products LIMIT 5; 返回不多于5行；
```
Comment (weiwensangsang) [inline Learning/Day3.md:67]: fix it.

File: Learning/Day3.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+
+SHOW CREATE DATABASE和SHOW CREATE TABLE，分别用来显示创建特定数据库或表的MySQL语句;
+
+SHOW GRANTS，用来显示授予用户(所有用户或特定用户)的安全权限;
+
+SHOW ERRORS和SHOW WARNINGS，用来显示服务器错误或警告消息。
+
+#### SELECT
+
+SELECT prod_name FROM products;
```
Comment (weiwensangsang) [inline Learning/Day3.md:58]: fix it.

File: Learning/Day3.md [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+
+周二上午10点
+读书笔记：各种指令的用法 select create update from (至少前15章)
+在mysql上用指令建立一个表
+
+# 数据库读书笔记
+
+### 数据库基础
+
+**数据库(database)** 保存有组织的数据的容器(通常是一个文件或一组文件)。
```
Comment (weiwensangsang) [inline Learning/Day3.md:16]: 这是一个错误，修复。

File: Learning/Day3.md [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+
+**表(table)** 某种特定类型数据的结构化清单。
```
Comment (weiwensangsang) [inline Learning/Day3.md:18]: 我觉得这个可指的探讨。

File: Learning/Day3.md (hunk 1 above)
Comment (whtsogreat) [reply Learning/Day3.md:7]: fix it.

**Pull request merged**